### ISSUE TITLE
Error in saveWorkbook

### ISSUE THREAD

**Issue by samuel-rosa** [ycphs/openxlsx#60]
I am loading a public workbook available at https://docs.google.com/spreadsheets/d/1rXIiT1zSYhFegSdAvE0yJX16q-bvXVNpYIYdd5YgjhI/export?format=xlsx. Loading is successful. However, I am not able to save it. The following error message is issued:

`Error in comment_list[[i]]$style[[j]] : subscript out of bounds`

The code that I am using is below.

```r
wb <- openxlsx::loadWorkbook(
   file = "https://docs.google.com/spreadsheets/d/1rXIiT1zSYhFegSdAvE0yJX16q-bvXVNpYIYdd5YgjhI/export?format=xlsx")
openxlsx::saveWorkbook(wb = wb, file = 'tmp-febr-planilha-modelo-geral.xlsx', overwrite = T)
```

**Comment by samuel-rosa** [comment]
It appears that the problem is due to the presence of notes in the cells.

**Comment by samuel-rosa** [comment]
So the error is due to the fact that, in Google Sheets, one can enter notes and comments. `openxlsx` deals when with comments, but not with notes.

I will close this because I found a solution for my problem.

**Comment by samuel-rosa** [comment]
Just noticed that `openxlsx` does not export comments in XLSX files generated using Google Sheets.

**Comment by samuel-rosa** [comment]
I have been working to understand and solve this issue. It appears that `openxlsx` requires all worksheets to contain comments.

I looked at the error message above and started looking into the structure of comments in my workbook using `wb$comments`. It is a list of lists, one list for each worksheets. If a worksheet does not contain any comment, then it is an empty list. I also noticed that Google Sheets did not export the `style` element of the note.

The solution I found to solve the error above was to add a note to all worksheets plus enforce an style using the following:

```r
for (i in 1:length(wb$comments)) {
  for (j in 1:length(wb$comments[[i]])) {
    wb$comments[[i]][[j]]$style <- '"<rPr><sz val=\"10\"/><color rgb=\"FF000000\"/><rFont val=\"Arial\"/><family val=\"0\"/><charset val=\"1\"/></rPr>"'
  }
}
```
